winui/Kanban/columns.md

```diff
@@ -0,0 +1,1339 @@
+---
+layout: post
+title: Columns in WinUI Kanban control | Syncfusion
+description: Learn here all about configuring and customizing columns in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Columns in WinUI Kanban (SfKanban) control
+
+The WinUI Kanban control organizes its layout into columns, each representing a distinct stage of a workflow, such as to-do, in progress, testing, validation, and completed. It supports various operations, including drag-and-drop, swim lane, and column toggling, all based on the defined columns.
+
+## Change column width
+
+By default, the columns are sized smartly to arrange the default elements of the cards with better readability. You can also define the minimum and maximum widths for the columns in `SfKanban` using the `MinimumColumnWidth` and `MaximumColumnWidth` properties, respectively.
```

Comment:
By default, the columns are sized smartly to arrange the default UI of the cards with better readability.

Reply:
Content changed